Bio/Seq.py

```diff
@@ -1392,7 +1392,7 @@ def count_overlap(self, sub, start=0, end=sys.maxsize):
         return end - start - len_sub_str + 1
 
     def complement(self):
-        """Return the complement of an unknown nucleotide equals itself.
+        """Return the complement of an unknown nucleotide usually equals itself.
```

Comment:
"Return the sequence complement"

Reply:
Yes, this is what I meant by the docstrings needing a bit more work.